worker/task.go

```diff
@@ -2422,7 +2437,10 @@ func (qs *queryState) handleHasFunction(ctx context.Context, q *pb.Query, out *p
 			return nil
 		}
 
-		_, err := qs.getValsForUID(q.Attr, lang, uid, q.ReadTs)
+		_, sz, err := qs.getValsForUID(q.Attr, lang, uid, q.ReadTs)
+		if err == nil {
+			out.ReadBytes += sz
```

Comment:
getValsForUid returns sz=0 in case of errors. You could directly do `out.ReadBytes += sz` without checking the error.

Reply:
Done.